sparse/src/KokkosSparse_par_ilut.hpp

```diff
@@ -165,6 +176,28 @@ void par_ilut_symbolic(KernelHandle* handle, ARowMapType& A_rowmap,
 
 }  // par_ilut_symbolic
 
+/// @brief
```

Comment:
And here's the dox for this one:

```
Performs the numeric phase (for specific CSRs, not reusable) of the par_ilut algorithm (described in the header). It is expected that par_ilut_symbolic has already been called for the provided KernelHandle.

/// @tparam KernelHandle Template for the handle type
/// @tparam ARowMapType Template for the A_rowmap type
/// @tparam AEntriesType Template for the A_entries type
/// @tparam AValuesType Template for the A_values type
/// @tparam LRowMapType Template for the L_rowmap type
/// @tparam LEntriesType Template for the L_entries type
/// @tparam LValuesType Template for the L_values type
/// @tparam URowMapType Template for the U_rowmap type
/// @tparam UEntriesType Template for the U_entries type
/// @tparam UValuesType Template for the U_values type
/// @param handle The kernel handle. It is expected that create_par_ilut_handle has been called on it
/// @param A_rowmap The row map (row nnz offsets) for the A CSR (Input)
/// @param A_entries The entries (column ids) for the A CSR (Input)
/// @param A_values The values (non-zero matrix values) for the A CSR (Input)
/// @param L_rowmap The row map (row nnz offsets) for the L CSR (Input/Output)
/// @param L_entries The entries (column ids) for the L CSR (Output)
/// @param L_values The values (non-zero matrix values) for the L CSR (Output)
/// @param U_rowmap The row map (row nnz offsets) for the U CSR (Input/Output)
/// @param U_entries The entries (column ids) for the U CSR (Output)
/// @param U_values The values (non-zero matrix values) for the U CSR (Output)
/// @param deterministic Run the compute LU factors part of the algorithm in serial. Allows for repeatable results but at a big performance penalty.
```

Reply:
I am leaving the deterministic arg undocumented since it is going away as soon as my latest par_ilut PR goes in.